pkg/core/blockchain.go

```diff
@@ -1104,7 +1107,10 @@ func (bc *Blockchain) verifyHashAgainstScript(hash util.Uint160, witness *transa
 	vm.RegisterInteropFuncs(interopCtx.getNeoInteropMap())
 	vm.LoadScript(verification)
 	vm.LoadScript(witness.InvocationScript)
-	vm.Run()
+	_, err := vm.Run()
+	if err != nil {
+		log.WithError(err)
```

Comment:
Same thing here, but three lines below. I don't think there is a case for `err != nil && !vm.HasFailed()`

Reply:
there were 2 cases. But set up there fault state since it looks logical